externals/kyuubi-spark-monitor/pom.xml

```diff
@@ -32,6 +32,12 @@
     <name>Kyuubi Project Spark Monitor</name>
 
     <dependencies>
+
+        <dependency>
+            <groupId>org.scala-lang</groupId>
+            <artifactId>scala-library</artifactId>
```

Comment:
`kyuubi-common` depends on `scala-library`, seems no require to declare again?

Reply:
removed